resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -4,15 +4,25 @@ set -e
 restartfile=<%= EZBake::Config[:restart_file] %>
 timeout=<%= EZBake::Config[:start_timeout] %>
 dir=$(dirname "$restartfile")
+PIDFILE=/var/run/puppetlabs/<%= EZBake::Config[:real_name] %>/<%= EZBake::Config[:real_name] %>
 
-if [ ! -r "$restartfile" ] || [ ! -w "$restartfile" ]; then
-    echo "The restart-file at <%= EZBake::Config[:restart_file] %> is not readable and/or writeable."
-    exit 1
-fi
+find_my_pid() {
+    pid=`pgrep -f <%= EZBake::Config[:uberjar_name] %>`
+    if [ ! -d  "/var/run/puppetlabs/${realname}" ] ; then
+        mkdir -p /var/run/puppetlabs/${realname}
+        chown -R $USER:$GROUP /var/run/puppetlabs/${realname}
+    fi
+    echo $pid > $PIDFILE
+}
 
 if [ ! -e "$restartfile" ]; then
     mkdir -p "$dir"
     echo "1" > "$restartfile"
+else
+    if [ ! -r "$restartfile" ] || [ ! -w "$restartfile" ]; then
```

Comment:
Can lines 21 and 22 be combined into `elif [ ! -r "$restartfile" ] || [ ! -w "$restartfile" ]; then`?